recipe/meta.yaml

```diff
@@ -1,38 +1,49 @@
 {% set name = "dash_cytoscape" %}
-{% set version = "0.2.0" %}
+{% set version = "0.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0669c79c197e4b150a5db7a278d1c7acebc947f3f5cbad5274835ebb44f712cd
+  sha256: a71ad4fe095570b71d4ad7c0d29199e9780c2e6796173d3b25fccc2cc58c855f
+  patches:
+    - 0001-point-to-correct-file-location.patch
 
 build:
-  noarch: python
-  number: 1
-  script: '{{ PYTHON }} -m pip install . -vv '
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch  # [unix]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.5
+    - setuptools
+    - wheel
+    - python
   run:
     - dash
-    - python >=3.5
+    - python
 
```

Comment:
some more requierements are needed, see https://github.com/plotly/dash-cytoscape/blob/v0.3.0/requirements.txt ?
biopython>=1.77
gunicorn
plotly
colour==0.1.5
?

Reply:
I am not sure. setup.py does not use requirements.txt file. In setup.py, only unpinned dash is mentioned:

https://github.com/plotly/dash-cytoscape/blob/v0.3.0/setup.py#L25

Reply:
it is inconsistent but used on their CI
https://github.com/plotly/dash-cytoscape/blob/29543d07d3ae459eb56bb809e3a36f7aa7b3072c/.circleci/config.yml#L66

Reply:
gunicorn not used anywhere. 
plotly will be pulled in by dash itself.

Reply:
@M-Waszkiewicz-Anaconda note that their CI installs `tests/requirements.txt`, which is a requirements file for their tests.